airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -468,6 +480,21 @@ def execute(self, context: Context):
             "serviceRole": self.service_role,
             "tags": self.tags,
         }
-        self.hook.client.create_compute_environment(**trim_none_values(kwargs))
+        response = self.hook.client.create_compute_environment(**trim_none_values(kwargs))
+        arn = response["computeEnvironmentArn"]
```

Comment:
If you move this one inside the if statement

Reply:
ah yes, I wanted to return it because I thought it was a nice addition, but then I forgot to.